README.md

```diff
@@ -58,12 +58,20 @@ Some of the services included so far:
 
 ## Getting started
 
+<a href="https://repology.org/metapackage/polybar">
```

Comment:
There's a purpose for using HTML here: only with HTML it's possible to use ```align="right"``` attribute which makes the badge right-aligned and floating, so it doesn't create a gap in text, [see how it looks](https://github.com/AMDmi3/polybar/tree/more-packages#getting-started), which IMO is the best way of embedding it. If you don't like it I can turn it into markdown (thus make it non floating), a plain text link, or remove it completely.